components/NavBar/NavBar.styled.ts

```diff
@@ -20,8 +20,6 @@ export const Background = styled.section`
   background: white;
   border-bottom: 1px solid #e8ecfd;
   z-index: 2;
-  position: fixed;
```

Comment:
We should not remove this styling. Removing `position: fixed` and `top: 0` introduces this bug:

### Without styling

<img width="496" alt="Screen Shot 2021-03-23 at 10 51 31 AM" src="https://user-images.githubusercontent.com/32081352/112194361-00e93100-8bc6-11eb-8240-7768bfca262e.png">


### With styling (before)

<img width="497" alt="Screen Shot 2021-03-23 at 10 52 43 AM" src="https://user-images.githubusercontent.com/32081352/112194373-03e42180-8bc6-11eb-9996-1de196d65e13.png">